autoload/neomake/makers/ft/d.vim

```diff
@@ -49,8 +49,7 @@ function! neomake#makers#ft#d#dmd()
     return {
         \ 'args': l:args,
         \ 'errorformat':
-        \     '%-G%f:%s:,%f(%l): %m,' .
-        \     '%f:%l: %m',
+        \     '%f(%l\,%c): %m,'
```

Comment:
The error format that was there was shamelessly copied from syntastic, which appears to be taken from a very old forum post dating ~2004 after a bit of googling.

The error format I replaced it with works just fine(actually, a bit better - the original had an issue parsing template errors.)

But you are right on the vcolumns point, I was under the impression that the other compilers adhered strictly to the reference compiler's flags, but was wrong. It appears that the GCC-based compiler doesn't even output the same format(I didn't have it installed when I first wrote it,) but in GCC format.

Going to push a few changes to fix all of this.